motif-docs/pages/guides/advanced/deploying-own-image.mdx

```diff
@@ -0,0 +1,51 @@
+---
+heading: Beyond the basics
+title: Deploying Your Own Image
+description: Using Image Registries to deploy a pre-built image
+noTOC: false
+---
+
+import Image from 'next/image'
+
+import imageRegistryGuiPic from '../../../public/img/image-registry-gui.png'
+import imageRegistryTablePic from '../../../public/img/image-registry-table.png'
+
+# Introduction
+
+Flightcontrol allows you to deploy your own pre-built image with an image registry, which allows you to use the platform as CD, instead of only CI/CD. To clarify our terminology, we use "registry" to refer to the host service that stores a collection of image repostorities, which we refer to as a "repository."
+The intended use of this feature is to support private repositories.
+Image registries store the credentials or details to a registry, which you can then reference later for using a pre-built image in a Fargate service.
+
+# Creating an Image Registry
```

Comment:
I think we should not use language to "create registry", because we are not creating a registry, rather linking one to flightcontrol system.

```suggestion
# Add an Image Registry to Flightcontrol
```